hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Schedulable.java

```diff
@@ -29,5 +29,9 @@
 public interface Schedulable {
   public UserGroupInformation getUserGroupInformation();
 
+  default CallerContext getCallerContext() {
+    return null;
```

Comment:
`throw new NotImplementedException("Code is not implemented");` instead?

Reply:
Done!